man/shadow.5.xml

```diff
@@ -137,12 +136,11 @@
 	<listitem>
 	  <para>
 	    The maximum password age is the number of days after which the
-	    user will have to change her password.
+	    user must change their password.
 	  </para>
 	  <para>
-	    After this number of days is elapsed, the password may still
-	    be valid.  The user should be asked to change her password the
-	    next time she will log in.
+	    After this number of days has elapsed, the password may still be valid.
+	    The user is prompted to change their password at the next login.
```

Comment:
@hallyn , @ikerexxe 

I will propose deprecation of this field eventually.  Security guidelines these days say that expiring passwords results in worse passwords.